recipes/agg-regrid/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "agg-regrid" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "1ab034cd9c5363afcd2cf74073bf11ab16d9e3c81729433eb008fcb6e81be976" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/SciTools-incubator/iris-{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [osx]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
+  run:
+    - python
+    - iris
+
+test:
+  imports:
+    - agg_regrid
+
+about:
+  home: https://github.com/SciTools-incubator/iris-{{ name }}
+  license: BSD-3-clause
+  license_family: BSD
+  license_file: COPYING
+  summary: "A conservative area-weighted regridder that uses AGG to rasterise a source iris cube to a target grid"
+
+extra:
+  recipe-maintainers:
+    - bjlittle
+    - pelson
+    - cpelley
```

Comment:
@cpelley, are you happy to co-maintain `agg-regrid`?

Reply:
Sure, thanks for checking :)